### PR TITLE
User can choose hotel

### PR DESCRIPTION
**Pull request by mantanquzzmo**
[PT-Link](https://www.pivotaltracker.com/story/show/171183949)

As a user
In order to add a hotel to my trip
I want to be able to chose my hotel

- adds index action
- adds destroy action

### CODE REVIEW

File: spec/requests/api/v1/hotels/delete_spec.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    let!(:other_trip) { create(:trip) }
+    3.times do
+      let!(:trip_hotels) { create_list(:hotel, 3, trip_id: trip.id) }
+      let!(:other_trip_hotels) { create_list(:hotel, 3, trip_id: other_trip.id) }
+    end
+
+    it 'that are not chosen are deleted' do
+      delete '/api/v1/hotels',
+             params: { hotel_id: trip.hotels.first.id,
+            trip: trip.id }
```
Comment (sundqvistjohan) [inline spec/requests/api/v1/hotels/delete_spec.rb]: Indentation

File: spec/requests/api/v1/hotels/delete_spec.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            
+      expect(trip.hotels.length).to eq 1
+    end
+
+    it 'that belong to other trips are not deleted' do
+      delete '/api/v1/hotels',
+             params: { hotel_id: trip.hotels.first.id,
+              trip: trip.id }
```
Comment (sundqvistjohan) [inline spec/requests/api/v1/hotels/delete_spec.rb]: Indentation

**Review by sundqvistjohan — CHANGES_REQUESTED** [review]
Updates look good to me, some small formatting issues

File: spec/requests/api/v1/hotels/delete_spec.rb (hunk 1 above)
Comment (mantanquzzmo) [reply spec/requests/api/v1/hotels/delete_spec.rb]: done

File: spec/requests/api/v1/hotels/delete_spec.rb (hunk 2 above)
Comment (mantanquzzmo) [reply spec/requests/api/v1/hotels/delete_spec.rb]: done

**Review by sundqvistjohan — APPROVED** [review]

**Comment by EevanR** [comment]
LGTM!

**Pull request merged**